File: Java/infixtopostfix.java
// Infix to Postfix Conversion using Stack in Java

import java.util.Stack;

public class infixtopostfix {
    // Infix expressio
    static String exp = "a+b*(c^d-e)^(f+g*h)-i";
    // Stack
    static Stack<Character> stack = new Stack<Character>();

    public static void main(String[] args) {
        // Infix to Postfix Conversion using Stack
        infixToPostfix(exp);
    }

    static void infixToPostfix(String exp) {
        // Infix to Postfix Conversion using Stack
        // your code here

    }

}

// Output:
// abcd^e-fgh*+^*+i-

File: Go/infixtopostfix.go
// Infix to Postfix Conversion using Stack in Golang

package main

import (
	"fmt"
)

func main() {
	var infix string = "a+b*(c^d-e)^(f+g*h)-i"
	fmt.Println("Infix Expression: ", infix)
	fmt.Println("Postfix Expression: ", infixToPostfix(infix))
}

func infixToPostfix(infix string) string {
	// your code here
	return "" // change this to return the postfix expression
}

// Output:
// Infix Expression:  a+b*(c^d-e)^(f+g*h)-i
// Postfix Expression:  abcd^e-fgh*+^*+i-
